enterprise_subsidy/apps/transaction/management/commands/write_reversals_from_enterprise_unenrollments.py

```diff
@@ -126,6 +126,13 @@ def handle_reversing_enterprise_course_unenrollment(self, unenrollment):
             )
             return 0
 
+        if related_transaction.state != TransactionStateChoices.COMMITTED:
+            logger.info(
+                f"{self.dry_run_prefix}Transaction: {related_transaction} is not in a committed state. "
+                f"Skipping Reversal creation."
+            )
+            return 0
```

Comment:
hm, since we're changing the behavior of `reverse_full_transaction()` in the openedx-ledger PR, and that same function is called in this function below, does that make this check somewhat redundant?  I'm generally in favor of keeping it since it causes an early termination, bypassing all the subsequent checks, but maybe add a comment to the code along the lines of "# Fail early if the transaction is not committed, even though reverse_full_transaction() would throw an exception later anyway.".

Reply:
Yes that's a great point! I'll add a comment.